mirage/config.js

```diff
@@ -705,6 +705,29 @@ function routes() {
     });
   });
 
-  this.patch('/users/:id');
+  this.patch('/users/:id', function (schema, request) {
+    const { id } = request.params;
+    const attrs = this.normalizedRequestAttrs();
+    const hasAnonymousModeEnabled = attrs.hasAnonymousModeEnabled;
```

Comment:
This is pointless indirection - why not use just `attrs.hasAnonymousModeEnabled` directly? Same for `githubUsername` below.